File: tests/fixtures/parity/shapes.kt
fun flatCalls(): Int {
    val a = 1
    val b = 2
    val c = a + b
    return c
}

fun pickBranch(x: Int): Int {
    if (x > 10) {
        return 3
    } else if (x > 5) {
        return 2
    } else {
        return 1
    }
}

fun nestedGuard(a: Int, b: Int, c: Int): Int {
    if (a > 0) {
        if (b > 0) {
            if (c > 0) {
                return 1
            }
        }
    }
    return 0
}

fun loopFilter(n: Int): Int {
    var total = 0
    for (i in 0 until n) {
        if (i % 2 == 0) {
            total += i
        }
    }
    return total
}

fun wideParams(a: Int, b: Int, c: Int, d: Int, e: Int, f: Int): Int {
    return a + b + c + d + e + f
}

fun boolBlend(a: Int, b: Int, c: Int, d: Int): Int {
    if ((a > 0 && b > 0) || (c > 0 && d > 0)) {
        return 1
    }
    return 0
}

File: tests/fixtures/parity/shapes.py
def flat_calls():
    a = 1
    b = 2
    c = a + b
    return c


def pick_branch(x):
    if x > 10:
        return 3
    elif x > 5:
        return 2
    else:
        return 1


def nested_guard(a, b, c):
    if a > 0:
        if b > 0:
            if c > 0:
                return 1
    return 0


def loop_filter(n):
    total = 0
    for i in range(n):
        if i % 2 == 0:
            total += i
    return total


def wide_params(a, b, c, d, e, f):
    return a + b + c + d + e + f


def bool_blend(a, b, c, d):
    if (a > 0 and b > 0) or (c > 0 and d > 0):
        return 1
    return 0
